primitives/src/vid/advz/payload_prover.rs

```diff
@@ -0,0 +1,505 @@
+// Copyright (c) 2023 Espresso Systems (espressosys.com)
+// This file is part of the Jellyfish library.
+
+// You should have received a copy of the MIT License
+// along with the Jellyfish library. If not, see <https://mit-license.org/>.
+
+//! Implementations of [`PayloadProver`] for `Advz`.
+//!
+//! Two implementations:
```

Comment:
It would be nice to give type aliases for these different proof types in terms of the high level properties/use cases, that make the contrasts obvious. I don't have any great ideas, though. Maybe something like `GranularPayloadProof`/`RangePayloadProof` or something?

Reply:
Agree, it's better to distinguish these two types of implementations/APIs.

Reply:
Renamed to `SmallRangeProof`, `LargeRangeProof` and tidied in https://github.com/EspressoSystems/jellyfish/pull/389/commits/eb8fde47d2e88a7dd27c2e8f88c9df6e75f2564d .

Unfortunately it's not clear how to give a useful type alias, as I would need to specify the type parameter `P`. But the whole file is generic over `P` so any type alias would need to hard-code a specific choice for `P`, which I don't want to do.

Reply:
You could do `type SmallRangeProof<P> = Proof<<P as Trait>::Proof>`. But renaming is also good 👍